core-web/libs/sdk/client/src/lib/client/content/shared/types.ts

```diff
@@ -0,0 +1,62 @@
+import { Equals } from '../../../query-builder/lucene-syntax/Equals';
+
+// Possible order values
+export type Order = 'asc' | 'desc';
+
+// Model to sort by fields
+export type SortBy = {
+    field: string;
+    order: Order;
+};
+
+export type SortByArray = Array<SortBy>;
+
+// Callback to build a query
+export type QueryBuilderCallback = (qb: Equals) => Equals;
```

Comment:
I am not sure about this, becuase this means that I can do something like:

```
(q) => q.build()
```

At least the editor think so.

Reply:
They cannot, it will throw an error in the method